app/controller/command/commands/__init__.py

```diff
@@ -3,5 +3,6 @@
 from app.controller.command.commands.team import TeamCommand
 from app.controller.command.commands.user import UserCommand
 from app.controller.command.commands.token import TokenCommand
+from app.controller.command.commands.project import ProjectCommand
 
-UnionCommands = Union[TeamCommand, UserCommand, TokenCommand]
+UnionCommands = Union[TeamCommand, UserCommand, TokenCommand, ProjectCommand]
```

Comment:
Should have no need for a union. We can use the parent class instead.

Reply:
Should this be addressed in another ticket? This change is only to address models.

Reply:
sure. a smaller ticket. very welcome